client/components/navbar.js

```diff
@@ -63,17 +63,34 @@ class Navbarclass extends Component {
                     onClick={() => {
                       handleClick()
                     }}
+                    className="linksInNavBar"
                   >
                     Logout
                   </Nav.Link>
                 </Nav>
               ) : (
                 <Nav className="mr-auto">
                   {/* This nav links will show these links if you are not logged in */}
-                  <Nav.Link as={Link} to="/login">
+                  <Nav.Link
+                    as={Link}
+                    to="/login"
+                    style={{
+                      color: 'black',
```

Comment:
Hey @aleksandrkayner! Could we actually make this dark blue: `#38495E`. That would match the color scheme from the logo!